src/Sulu/Bundle/CoreBundle/Resources/config/content.xml

```diff
@@ -61,6 +63,18 @@
             <argument type="service" id="sulu_content.compat.structure.legacy_property_factory" />
         </service>
 
+        <service id="sulu.content.webspace_structure_provider.cache" class="%sulu.content.webspace_structure_provider.cache.class%">
+            <argument type="string">%sulu.cache_dir%/webspace_structures</argument>
+        </service>
+
+        <service id="sulu.content.webspace_structure_provider" class="%sulu.content.webspace_structure_provider.class%">
+            <argument type="service" id="twig"/>
+            <argument type="service" id="sulu.content.structure_manager"/>
```

Comment:
Note that this is deprecated. We shoud be using the `structure.metadata_factory` or the `document_manager` depending on the use case.